### PR TITLE
Day15 Question

### PR DESCRIPTION
**Pull request by haingo94**
#### Related issues (Optional)

Task #70

#### Description (Required)
- [x] Create viewpager contain question fragments
- [x] Read and display data from file json
- [x] Handle button next and previous
- [x] List correct and incorrect answers
- [x] Use DialogFragment to exit and show result list
- [x] Random questions
#### Reference (Optional)

### CODE REVIEW

**Comment by at-tamnguyen2** [comment]
@haingo94 
LGTM
Thank you

**Comment by haingo94** [comment]
@at-tamnguyen2 thank you very much

File: app/src/main/java/vn/asiantech/internship/models/Result.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import android.os.Parcel;
+import android.os.Parcelable;
+
+/**
+ * Created by Hai on 6/25/2017.
+ */
+
+public class Result implements Parcelable {
+    private int questionId;
+    private boolean isCorrect;
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/internship/models/Result.java]: WARNING: [AvoidFieldNameMatchingMethodName] Field isCorrect has the same name as a method
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#AvoidFieldNameMatchingMethodName

File: app/src/main/res/layout/fragment_question_dialog.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        android:text="@string/question.dialog.title"
+        android:textSize="@dimen/question.dialog.textsize" />
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/tvDialogTitle"
+        android:gravity="center">
+
+        <Button
```
Comment (at-dev-support-01) [inline app/src/main/res/layout/fragment_question_dialog.xml]: WARNING: [ButtonStyle] Buttons in button bars should be borderless; use `style="?android:attr/buttonBarButtonStyle"` (and `?android:attr/buttonBarStyle` on the parent)
 Button bars typically use a borderless style for the buttons. Set the `style="?android:attr/buttonBarButtonStyle"` attribute on each of the buttons, and set `style="?android:attr/buttonBarStyle"` on the parent layout

File: app/src/main/res/layout/fragment_question_dialog.xml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+            android:id="@+id/btnCancel"
+            android:layout_width="100dp"
+            android:layout_height="wrap_content"
+            android:text="@string/question.dialog.cancel" />
+
+        <Button
```
Comment (at-dev-support-01) [inline app/src/main/res/layout/fragment_question_dialog.xml]: WARNING: [ButtonStyle] Buttons in button bars should be borderless; use `style="?android:attr/buttonBarButtonStyle"` (and `?android:attr/buttonBarStyle` on the parent)
 Button bars typically use a borderless style for the buttons. Set the `style="?android:attr/buttonBarButtonStyle"` attribute on each of the buttons, and set `style="?android:attr/buttonBarStyle"` on the parent layout

File: app/src/main/res/layout/fragment_result_dialog.xml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        android:text="@string/question.dialog.result.title"
+        android:textSize="@dimen/question.dialog.textsize" />
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/tvDialogTitle"
+        android:gravity="center">
+
+        <Button
```
Comment (at-dev-support-01) [inline app/src/main/res/layout/fragment_result_dialog.xml]: WARNING: [ButtonStyle] Buttons in button bars should be borderless; use `style="?android:attr/buttonBarButtonStyle"` (and `?android:attr/buttonBarStyle` on the parent)
 Button bars typically use a borderless style for the buttons. Set the `style="?android:attr/buttonBarButtonStyle"` attribute on each of the buttons, and set `style="?android:attr/buttonBarStyle"` on the parent layout

File: app/src/main/res/layout/fragment_result_dialog.xml [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+            android:id="@+id/btnResultCancel"
+            android:layout_width="100dp"
+            android:layout_height="wrap_content"
+            android:text="@string/question.dialog.cancel" />
+
+        <Button
```
Comment (at-dev-support-01) [inline app/src/main/res/layout/fragment_result_dialog.xml]: WARNING: [ButtonStyle] Buttons in button bars should be borderless; use `style="?android:attr/buttonBarButtonStyle"` (and `?android:attr/buttonBarStyle` on the parent)
 Button bars typically use a borderless style for the buttons. Set the `style="?android:attr/buttonBarButtonStyle"` attribute on each of the buttons, and set `style="?android:attr/buttonBarStyle"` on the parent layout

**Comment by at-tienhoang** [comment]
@haingo94 Please resolve conflicts

**Comment by at-tienhoang** [comment]
@haingo94 Please resolve conflict, thanks

**Comment by at-tienhoang** [comment]
LGTM, thanks

**Pull request merged**